Add input device stubs (#3411)
ppy/osu-wiki: wiki/Gameplay/Input_device/Wiimote/en.md

# Wiimote

> The **Wiimote** is the primary game controller for the Nintendo Wii console.  
> The Wiimote can be used in osu! to aim by using the sensor bar and you can rebind any button to click in the osu! options.
> Be sure to first enable support for it in the osu! options. 

Reviewer: ```suggestion
The **Wiimote** is the primary game controller for the Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Although seen as mostly gimmicky or silly, the Wiimote can be used in osu!standard to aim the cursor by using the sensor bar. can rebind any button to click in the osu! options.

The Wiimote may also be used to control Taiko drum hits via motion controls or buttons on the Wiimote itself. Though this may require some more advanced knowledge of software and Wiimotes in order to get working.
```
Taiko should really be linked to something. **This stub needs images.**

> ![Wiimote support option in osu!](img/1.jpg "Wiimote support option in osu!")

Reviewer: This image doesn't work; link leads to 404 error

Author: works for me, it did give 404 when looking from the pr but you can try looking from en.md instead of pr.

# Wiimote

> The **Wiimote** is the primary game controller for the Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). 
> Although seen as mostly gimmicky or silly, the Wiimote can be used in osu!standard to aim the cursor by using the sensor bar. 
> You can rebind any button to click in the osu! options.

Reviewer: ```suggestion
The **Wiimote** is the primary game controller for the Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Although seen as mostly gimmicky or silly, the Wiimote can be used in osu!standard to aim the cursor by using the sensor bar. You can rebind any button to click in the osu! options.
```

> The Wiimote may also be used to control Taiko drum hits via motion controls or buttons on the Wiimote itself. 
> Though this may require some more advanced knowledge of software and Wiimotes in order to get working.

Reviewer: ```suggestion
The Wiimote may also be used to control Taiko drum hits via motion controls or buttons on the Wiimote itself. Alhough this may require some more advanced knowledge of software and Wiimotes in order to get working.
```
